common/src/main/java/com/lightstep/tracer/shared/AbstractTracer.java

```diff
@@ -259,9 +249,11 @@ private void disable() {
         doStopReporting();
 
         synchronized (mutex) {
-            if (transport != null) {
-                transport.close();
-                transport = null;
+            try {
+                client.shutdown();
+            } catch (InterruptedException e) {
```

Comment:
Thanks for catching this - I should probably handle the exception inside the client anyway.